setup.py

```diff
@@ -19,7 +17,7 @@
     requirements = []
     for req in f.read().splitlines():
         if req.startswith("git+"):
-            os.system('pip install '+req)
+            os.system('pip3 install ' + req)
```

Comment:
:+1: :+1: :+1: :+1: (I should really fix this), there's also the same problem in [anpy](https://github.com/regardscitoyens/anpy/)